src/wordSelector.ts

```diff
@@ -0,0 +1,11 @@
+export class WordSelector {
+    wordsToPickFrom = ['table', 'hello', 'sunshine', 'knowledge', 'interface'];
```

Comment:
Žodžiai tikrai negali būti čia, o turėtų nugulti tavo duombazėje (failas, sql, mongo ar in-memory nesvarbu)